packages/web-app/src/components/StupidPlayers.tsx

```diff
@@ -24,8 +28,13 @@ export const StupidPlayers = () => {
           <span>Username: {username}</span>
           <span>Balance: {balance}</span>
           <span>Is Admin: {isAdmin}</span>
+          
         </div>
+        
       ))}
+      <button onClick={
+        () => refetch()
+        }>refetch</button>
```

Comment:
Linter should have fixed this, I will look into why it didn't and push change to fix this in the future. Will also add linting script to base of repo